packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -225,6 +226,13 @@ class TextureLayer extends Layer {
   /// The identity of the backend texture.
   final int textureId;
 
+  /// When true the texture that will not be updated with new frames.
+  ///
+  /// This is used when resizing an embedded  Android views: When resizing an
```

Comment:
extra space and Android view~s~

Maybe - "When resizing ~an Android view~ "- otherwise it is a bit repetitive.

Reply:
done